packages/builder/src/link.js

```diff
@@ -49,12 +53,11 @@ try {
 const skpmConfig = getSkpmConfigFromPackageJSON(packageJSON)
 
 if (!skpmConfig.main) {
-  console.error(
-    `${chalk.red(
-      'error'
+  console.warn(
+    `${chalk.yellow(
+      'warning'
     )} Missing "skpm.main" fields in the package.json. Should point to the ".sketchplugin" file`
   )
-  process.exit(1)
```

Comment:
any reason not to throw here? The rest of the script is going to break without `skpmConfig.main`

Reply:
Since I am passing the plugin-name.sketchplugin as an argument, which is different than skpm.main, it feels like skpm.main should not be a sketch-plugin name. I can change it back but feels redundant and confusing.

Reply:
it isn't. If you don't pass it as an option and it's not specified, then we want to fail.

If you do specify it as an option, then `skpmConfig.main` will be equal to it